packages/ui/src/molecules/ProductTitle/ProductTitle.tsx

```diff
@@ -24,21 +24,23 @@ export type ProductTitleProps = Omit<HTMLAttributes<HTMLElement>, 'title'> & {
   testId?: string
 }
 
-function ProductTitle({ title, label, refTag= "Ref.: ",refNumber, testId= 'store-product-title', ...otherProps }: ProductTitleProps) {
-  return (
-    <header data-fs-product-title data-testid={testId} {...otherProps}>
-      <div data-fs-product-title-header>
-        {title}
-        {!!label && label}
-      </div>
+const ProductTitle = forwardRef<HTMLDivElement, ProductTitleProps>(
```

Comment:
The correct one is to use `HTMLElement` since the `header` tag is used.

To reference: [see the topic DOM header tag interface in Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)
--


<br class="Apple-interchange-newline">

```suggestion
const ProductTitle = forwardRef<HTMLElement, ProductTitleProps>(
```